tests/phpunit/agentic-commerce/class-wc-stripe-agentic-commerce-manual-approval-test.php

```
@@ -0,0 +1,353 @@
+				'filter_return'   => 'Blocked by custom rule.',
+				'expected_type'   => 'declined',
+				'expected_reason' => 'Blocked by custom rule.',
+			],
+			'filter approves a declined order'  => [
+				'product_args'    => [ 'stock_status' => 'outofstock' ],
+				'filter_return'   => null,
+				'expected_type'   => 'approved',
+				'expected_reason' => null,
+			],
+		];
+	}
```

PHPMD (2.15.0)

[error] The method filter_override_provider is not named in camelCase. (undefined)

(CamelCaseMethodName)